src/Servers/IIS/IIS/test/Common.FunctionalTests/FrebTests.cs

```diff
@@ -107,6 +97,8 @@ public async Task CheckFrebDisconnect()
             await result.HttpClient.RetryRequestAsync("/WaitingRequestCount", async message => await message.Content.ReadAsStringAsync() == "1");
         }
 
+        result.HttpClient.Dispose();
```

Comment:
We should probably do this will all of the tests, it would speed up running locally. These tests were taking 1 minute each now it's 3 seconds.

Reply:
Hrm I'll try to make the update to the tests in a way where it will be easy to remove it (optimistically when in some future time this regression is fixed)

Reply:
Is there some kind of keep alive on the http client side, something like https://makolyte.com/csharp-configuring-how-long-an-httpclient-connection-will-stay-open/#In_NET_Core

Reply:
It seems possible to disable connection pooling by setting the lifetime to TimeSpan.Zero. I'll try it.

Reply:
Works but I'll let you figure out how to apply it. I don't know if we should be changing the deployers directly or just these tests.

https://github.com/dotnet/aspnetcore/blob/4cdf42e78b7f56fc28ff4090a84674334e3694c2/src/Servers/IIS/IntegrationTesting.IIS/src/IISDeploymentResult.cs#L29

```
HttpClient = CreateClient(new SocketsHttpHandler() { PooledConnectionLifetime = TimeSpan.Zero });
```